archway/chain.json

```diff
@@ -29,42 +29,34 @@
       {
         "denom": "aarch"
       }
-    ]
+    ],
+    "lock_duration": {
+      "time": "1209600s"
+    }
   },
   "codebase": {
     "git_repo": "https://github.com/archway-network/archway",
     "recommended_version": "v1.0.0",
     "compatible_versions": [
       "v1.0.0"
     ],
-    "cosmos_sdk_version": "v0.45.16",
-    "cosmwasm_version": "v0.32.0",
-    "cosmwasm_enabled": true,
-    "ibc_go_version": "v4.3.1",
-    "cosmwasm_path": "$HOME/.example/data/wasm",
     "binaries": {
       "linux/amd64": "https://github.com/archway-network/archway/releases/download/v1.0.0/archwayd_linux_amd64",
       "linux/arm64": "https://github.com/archway-network/archway/releases/download/v1.0.0/archwayd_linux_arm64"
     },
+    "cosmos_sdk_version": "v0.45.16",
+    "tendermint_version": "v0.34.27",
+    "cosmwasm_version": "v0.32.0",
+    "cosmwasm_enabled": true,
+    "ibc_go_version": "v4.3.1",
     "genesis": {
-      "genesis_url": "https://github.com/archway-network/networks/blob/main/archway-1/genesis/genesis.json.gz"
+      "genesis_url": "https://github.com/archway-network/networks/raw/main/archway-1/genesis/genesis.json.gz"
     },
     "versions": [
       {
-        "name": "genesis",
-        "recommended_version": "v1.0.0",
```

Comment:
Can you please re-add this stuff (from recommended_version to binaries) back in please?

Reply:
FYI I added it in